packages/pigeon/CHANGELOG.md

```diff
@@ -1,10 +1,19 @@
+## 9.0.2
+
+* [java] Adds a `GeneratedApi.FlutterError` exception for passing custom error parameters (code, message, details).
+* [kotlin] Adds a `FlutterError` exception for passing custom error parameters (code, message, details).
```

Comment:
The README should also be updated to explain and demonstrate returning errors with and without these classes.